code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -35,4 +35,21 @@
 	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
 	heat_protection = HANDS
 	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
-	burn_state = FIRE_PROOF
\ No newline at end of file
+	burn_state = FIRE_PROOF
+
+/obj/item/clothing/gloves/bracer
+	name = "bone bracers"
+	desc = "For when you're expecting to get slapped on the wrist. Offers modest protection to your arms."
+	icon_state = "bracers"
+	item_state = "bracers"
+	item_color = null	//So they don't wash.
+	transfer_prints = TRUE
+	strip_delay = 40
+	put_on_delay = 20
+	body_parts_covered = ARMS
+	cold_protection = ARMS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	burn_state = FIRE_PROOF
+	armor = list(melee = 15, bullet = 35, laser = 35, energy = 20, bomb = 35, bio = 35, rad = 35) //Not like anything ever hits the arms anyways.

```

Comment:
The fact of the matter is this 

1. This item is pretty much exclusively going to be used by miners and lavaland natives, the hassle of getting a pair from a miner as any job that could capitalize on the "MAX ARMOR" benefit of the arms isn't even really worth the effort.

2. You cannot wear insulated gloves and bracers, or any regular gloves.

3. Nobody ever hits the arms, it's just a small chance.